scripts/travis-helpers.sh

```diff
@@ -17,8 +17,10 @@ set -e
 
 source scripts/helpers.sh
 
-export RUNDECK_BUILD_NUMBER="${TRAVIS_BUILD_NUMBER}"
+# export RUNDECK_BUILD_NUMBER="${TRAVIS_BUILD_NUMBER}"
+export RUNDECK_BUILD_NUMBER="3467"
```

Comment:
unintentional?

Reply:
Indeed. I need a better way to override that.